model/ActivityMonitoringService.php

```diff
@@ -67,8 +67,7 @@ protected function getLoginQueueLength()
         $activeDeliveryExecution = new GetActiveDeliveryExecution();
 
         $queue = __('Turned off');
-        $limit = $actionQueue->getLimits($activeDeliveryExecution);
-        if ($limit > 0) {
+        if ($actionQueue->isActionEnabled($activeDeliveryExecution)) {
             $queue = $actionQueue->getPosition($activeDeliveryExecution);
         }
```

Comment:
Maybe it would be better to move `$queue = __('Turned off');` to the `else` block to avoid translation call and value reassignment?
```
if ($actionQueue->isActionEnabled($activeDeliveryExecution)) {
    $queue = $actionQueue->getPosition($activeDeliveryExecution);
} else {
    $queue = __('Turned off');
}
```

Reply:
fixed